gulpfile.js

```diff
@@ -92,6 +92,17 @@ gulp.task('svg-individual', function () {
 gulp.task('svg', ['svg-sprite', 'svg-individual']);
 
 
+// BUILD DESIGN SYSTEM STYLES
+gulp.task('build-styles', function () {
+	return gulp.src('dist/styles.scss')
+		.pipe(scss())
+		.pipe(autoprefixer({
+			browsers: ['last 3 versions'],
```

Comment:
I see mercury is using different config for supported browsers: https://github.com/Wikia/mercury/blob/4eb062496b96876656dde7c29147582638a10727/front/auth/gulpfile.js#L45
Do we need to do something about it?